addons/resource.language.en_gb/resources/strings.po

```diff
@@ -15568,7 +15568,19 @@ msgctxt "#35050"
 msgid "Controller profiles"
 msgstr ""
 
-#empty strings from id 35051 to 35054
+#. Setting to test rumble
```

Comment:
not sure that it's obvious to translators that read this description out of context. They neither see the settings category this setting is in, nor that it's related to input devices. So I think the settings description should be a bit more descriptive and also incluude that "rumble" is related to input devices.

Maybe: `Label of a button to test the rumble feature of input devices like game controllers` ?